chrome/content.js

```diff
@@ -3,25 +3,41 @@ $(function() {
 
 	$( document ).tooltip();
 
-	chrome.storage.sync.get('english-glossary', (data) => {
-		fetch(data['english-glossary'])
+	// get language
+	let language = 'english'
+	let langElement = $('.editorLoanLanguage')
+	if (langElement.length) {
+		let content = langElement.html()
+		language = content.substring(content.indexOf('<span>') + '<span>'.length, content.indexOf('<br>')).toLowerCase()
+	}
+
+	let glossaryName = language + '-glossary'
+
+	// get glossary
+	chrome.storage.sync.get(glossaryName, (data) => {
+		fetch(data[glossaryName])
 			.then((resp) => resp.json())
 			.then((json) => {
 				glossary = {...json[''], ...json[getCountry()]}
-				console.log('Finished loading translation glossary')
+				console.log(`Finished loading ${language} glossary`)
 				highlightTerms()
 			})
+			.catch((err) => {
+				console.log(`Could not load ${language} glossary`)
+				console.log(err)
+			})
 	})
 
-	// just an example of communication between content script and popup script
+	// gives the popup info about this page
 	chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
-		    if (request.enabled)
-		      	console.log(glossary)
+		    if (request.type == 'VIVA_PAGE_INFO') {
+		    	sendResponse({country: getCountry(), language: language})
+		    }
 	});
 
 	function getCountry() {
 		let location = $('a[href="#country-section"]').text()
-		return location.slice(location.indexOf(',') + 2)
+		return location.slice(location.lastIndexOf(',') + 2)
```

Comment:
oh interesting, does it perhaps sometimes occur twice, thus seek the lastIndexOf?